src/views/contact/components/ContactPageIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionComponent } from 'react';
+import { Flex, FlexProps, Icon, IconProps } from '@chakra-ui/react';
+
+type ContactPageIconProps = {
+  icon: FunctionComponent<IconProps>;
+} & FlexProps;
+
+const ContactPageIcon: FunctionComponent<ContactPageIconProps> = ({ icon, ...props }) => (
```

Comment:
I find it easier and cleaner to use it that way because I can reuse the component with different icons only by passing it as a prop. If you advice me to change it I will be more than happy to do so :)